src/views/mfa-verify/PushForm.js

```diff
@@ -29,6 +31,18 @@ define(['okta'], function (Okta) {
 
     initialize: function () {
       this.enabled = true;
+      this.listenTo(this.options.appState, 'change:isSuccessResponse',
```

Comment:
The problem is that this only works if you have no other states that happen after the mfa challenge - it's not "MFA SUCCESS", it's "Auth SUCCESS". So for example, if you're password has expired, this code won't work because this controller will already be removed from the DOM.

You can do this:
Create a trapAuthResponse function. This gets called before we navigate to the next controller. Set the cookie there, and return false.

More info here: https://github.com/okta/okta-signin-widget/blob/master/src/util/RouterUtil.js#L97